### PR TITLE
Update Makefile and go.yml

### PR DESCRIPTION
**Pull request by hapoon**

### CODE REVIEW

File: .github/workflows/go.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ jobs:
         fi
 
     - name: Build
-      run: go build -v .
+      run: go build -v cmd/hapoon/main.go
```
Comment (aoi31) [inline .github/workflows/go.yml]: Can't you use Makefile commands?

File: .github/workflows/go.yml [hunk 2]
```diff
@@ -31,7 +31,7 @@ jobs:
...
 
     - name: Test
-      run: go test -v .
+      run: go test -v ./...
```
Comment (aoi31) [inline .github/workflows/go.yml]: This too.

**Review by aoi31 — COMMENTED** [review]
I comment that I was interested.

**Comment by hapoon** [comment]
@aoi31 
Please review 🙏

File: .github/workflows/go.yml [hunk 3]
```diff
@@ -31,7 +31,7 @@ jobs:
         fi
 
     - name: Build
-      run: go build -v .
+      run: make build
```
Comment (aoi31) [inline .github/workflows/go.yml:34]: 👍

File: .github/workflows/go.yml [hunk 4]
```diff
@@ -31,7 +31,7 @@ jobs:
...
 
     - name: Test
-      run: go test -v .
+      run: make test
```
Comment (aoi31) [inline .github/workflows/go.yml:37]: 👍

**Review by aoi31 — APPROVED** [review]
LGTM

**Pull request merged**